File: app/src/main/java/com/dsm104desafiopractico2/clases/ListaProductos.java
package com.dsm104desafiopractico2.clases;

public class ListaProductos {
    String nombre;
    String imagen;
    double precio;

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getImagen() {
        return imagen;
    }

    public void setImagen(String imagen) {
        this.imagen = imagen;
    }

    public double getPrecio() {
        return precio;
    }

    public void setPrecio(double precio) {
        this.precio = precio;
    }

    public ListaProductos(String nombre, String imagen, double precio) {
        this.nombre = nombre;
        this.imagen = imagen;
        this.precio = precio;
    }

    public ListaProductos() {
    }
}

File: app/src/main/java/com/dsm104desafiopractico2/clases/ListaProductos.kt
package com.dsm104desafiopractico2.clases;

public class ListaProductos {
    var nombre:String? =null
    var id:String? =null
    var imagen:String? = null
    var precio:Double? = 0.0
    var tipo:String? = null

    constructor(){}

    constructor(nombre:String?,precio:Double?){
        this.nombre = nombre
        this.precio = precio
    }

    constructor(id:String?,nombre:String?,precio:Double?){
        this.id = id
        this.nombre = nombre
        this.precio = precio
    }

    constructor(id:String?,nombre:String?,precio:Double?,tipo:String?){
        this.id = id
        this.nombre = nombre
        this.precio = precio
        this.tipo = tipo
    }

}
